jerry-core/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ message(STATUS "MEM_HEAP_SIZE_KB          " ${MEM_HEAP_SIZE_KB})
 set(INCLUDE_CORE
     "${CMAKE_CURRENT_SOURCE_DIR}"
     "${CMAKE_CURRENT_SOURCE_DIR}/api"
-    "${CMAKE_CURRENT_SOURCE_DIR}/debugger"
+    "${CMAKE_CURRENT_SOURCE_DIR}/dbg"
```

Comment:
I thought that with prefix conflicts we go with the "cross that bridge when we come to it" approach?
> do nothing until this issue got hit by someone